deploy/deploy.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+command_exists() {
+    command -v "$@" > /dev/null 2>&1
+}
+
+install_docker() {
+    sudo apt-get update
+    sudo apt-get install \
+        ca-certificates \
+        curl \
+        gnupg \
+        lsb-release
+    sudo mkdir -p /etc/apt/keyrings
+    sudo rm /etc/apt/keyrings/docker.gpg || true
+    curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+    echo \
+        "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \
+        $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update -y
+    sudo apt-get install -y docker-ce=5:20.10.18~3-0~debian-buster docker-ce-cli=5:20.10.18~3-0~debian-buster containerd.io docker-compose-plugin
+}
+
+install_openresty() {
+    sudo apt-get -y install --no-install-recommends wget gnupg ca-certificates
+    wget -O - https://openresty.org/package/pubkey.gpg | sudo apt-key add -
+    codename=`grep -Po 'VERSION="[0-9]+ \(\K[^)]+' /etc/os-release`
+    echo "deb http://openresty.org/package/debian $codename openresty" \ | sudo tee /etc/apt/sources.list.d/openresty.list
+    sudo apt-get update
+    sudo apt-get -y install openresty
+}
+
+start_openresty() {
+    ulimit -n 65535
+    ulimit -n -S
+    ulimit -n -H
+    openresty -p ~/apisix-benchmark/deploy/openresty -c ~/apisix-benchmark/deploy/openresty/conf/nginx.conf
```

Comment:
We can avoid hardcoding the test path? Maybe we can use env var for it

Reply:
In fact, I hope to complete this optimization in the future, see the TODO in `Readme.md`.